kerastuner/utils.py

```diff
@@ -20,7 +20,7 @@
 
 def compute_model_size(model):
     "comput the size of a given model"
-    params = [K.count_params(p) for p in set(model.trainable_weights)]
+    params = [K.count_params(p) for p in model.trainable_weights]
```

Comment:
If you need to de-duplicate `model.trainable_weights`, use `id(x)` where x is a weight. Ids should be unique